charts/leonardo/values.yaml

```diff
@@ -43,7 +43,8 @@ cronjob:
   # deployments.standalone.name -- Name to use for the default standalone Leonardo deployment
   name: leonardo-resource-validator-cronjob
   imageRepository: us.gcr.io/broad-dsp-gcr-public/resource-validator
-  imageTag: b994ab5
+  imageTag: latest
```

Comment:
Should we peg to a specific hash instead?

Reply:
this needs to be updated once your PR is merged...I'm using `latest` temporarily because this set up won't work without https://github.com/broadinstitute/leonardo-cron-jobs/pull/17

Reply:
oh actually, broadinstitute/leonardo-cron-jobs#17 will generate new images as well..I'll update here once those images are published